templates/CRM/Admin/Page/Access.tpl

```diff
@@ -28,28 +28,14 @@
     <p>{ts 1=$docLink}ACLs (Access Control Lists) allow you control access to CiviCRM data. An ACL consists of an <strong>Operation</strong> (e.g. 'View' or 'Edit'), a <strong>set of Data</strong> that the operation can be performed on (e.g. a group of contacts), and a <strong>Role</strong> that has permission to do this operation. Refer to the %1 for more info.{/ts}
     {if $config->userSystem->is_drupal EQ '1'}{ts}Note that a CiviCRM ACL Role is not related to the Drupal Role.{/ts}{/if}</p>
     <p>{ts}<strong>EXAMPLE:</strong> 'Team Leaders' (<em>ACL Role</em>) can 'Edit' (<em>Operation</em>) all contacts in the 'Active Volunteers Group' (<em>Data</em>).{/ts}</p>
-    {if $config->userSystem->is_drupal EQ '1'}
-        <p>{ts 1=$ufAccessURL}Use <a href='%1'>Drupal Access Control</a> to manage basic access to CiviCRM components and menu items. Use CiviCRM ACLs to control access to specific CiviCRM contact groups. You can also configure ACLs to grant or deny access to specific Events Profiles, and/or Custom Data Fields.{/ts}</p>
-    {elseif $config->userFramework EQ 'Joomla'}
-        <p>{ts 1=$ufAccessURL 2=$jAccessParams}Use <a href='%1' %2>Joomla Access Control</a> to manage basic access to CiviCRM components and menu items. Use CiviCRM ACLs to control access to specific CiviCRM contact groups. You can also configure ACLs to grant or deny access to specific Events, Profiles, and/or Custom Data Fields.{/ts}</p>
-   {elseif $config->userFramework EQ 'WordPress'}
-        <p>{ts 1=$ufAccessURL}Use <a href='%1'>WordPress Access Control</a> to manage basic access to CiviCRM components and menu items. Use CiviCRM ACLs to control access to specific CiviCRM contact groups. You can also configure ACLs to grant or deny access to specific Events, Profiles, and/or Custom Data Fields.{/ts}</p>
-   {/if}
+    <p>{ts 1=$ufAccessURL 2=$jAccessParams 3=$config->userFramework}Use <a href='%1' %2>%3 Access Control</a> to manage basic access to CiviCRM components and menu items. Use CiviCRM ACLs to control access to specific CiviCRM contact groups. You can also configure ACLs to grant or deny access to specific Events, Profiles, and/or Custom Data Fields.{/ts}</p>
```

Comment:
I have mixed feelings about this, because it is exposing a machine-name that happens to be convenient in English, but given that it's the ACL admin form, I'm not opposed to the cleanup.

Also, the expression `%1 Access Control` is already present in that page, so this is not introducing a regression from the point of view of i18n.

You can ignore this:

- It's a form of concatenation which may not fit in all languages, although it should be OK in the languages I know, so maybe not a big deal.

- In non-latin languages, people may or may not translate the CMS name. For example, Друпал in Russian (although the current Russian translation of CiviCRM uses 'Drupal' and not 'Друпал', which may sound obvious, but McDonalds in Russia is Макдоналдс, for example).

- Does "Use Drupal8 Access Control" sound OK? (given that this is a geeky admin form, I don't really mind, but it is a bit inconsistent)

- I would have recommended to use "Content Management System Access Control", but I have to admit that it's both annoyingly long and vague.